userspace/falco/webserver.cpp

```diff
@@ -68,6 +69,28 @@ void falco_webserver::start(
             res.set_content(versions_json_str, "application/json");
         });
 
+    if (!metrics_collectors.empty())
+    {
+        libs::metrics::prometheus_metrics_converter prometheus_metrics_converter;
+
+        m_server->Get("/metrics",
+            [metrics_collectors, prometheus_metrics_converter](const httplib::Request &, httplib::Response &res) {
+                std::string prometheus_text;
+
+                for (auto metrics_collector: metrics_collectors) {
+                    metrics_collector.snapshot();
+                    auto metrics_snapshot = metrics_collector.get_metrics();
+
+                    for (auto& metric: metrics_snapshot)
+                    {
+                       prometheus_metrics_converter.convert_metric_to_unit_convention(metric);
+                       prometheus_text += prometheus_metrics_converter.convert_metric_to_text_prometheus(metric, "namespace", "falco") + "\n";
```

Comment:
Do we need the additional `+ "\n"` as the returned `prometheus_text` should already end with a new line.

Reply:
Indeed not, old string building muscle memory.